### PR TITLE
Readme updated to include Revit 2023

### LINKED ISSUE #1306: Update Readme to include Revit 2023
#### What is missing/incorrect?

As in the title, currently Readme says only Revit up to 2022 is supported.

### PR DESCRIPTION
**Pull request by pawelbaran**
### Issues addressed by this PR

Closes #1306

### Test files

Not needed

### CODE REVIEW

**Comment by pawelbaran** [comment]
@BHoMBot check compliance
@BHoMBot check core

**Comment by pawelbaran** [comment]
@BHoMBot check null-handling
@BHoMBot check serialisation
@BHoMBot check versioning
@BHoMBot check installer

File: README.md [hunk 1]
```diff
@@ -37,6 +38,7 @@ You will need the following to build Revit_Toolkit:
     - .NET Framework 4.7.2 for Revit 2020
```
Comment (vietle-bh) [inline README.md:38]: Should we change all this to .NET Framework 4.8?

Comment (pawelbaran) [reply README.md]: good point @vietle-bh - fixed 👍

**Review by vietle-bh — APPROVED** [review]
ReadMe file never looked so good 👍

**Comment by pawelbaran** [comment]
@BHoMBot check required

**Comment by pawelbaran** [comment]
@BHoMBot check ready-to-merge

**Comment by pawelbaran** [comment]
@BHoMBot check installer

**Comment by pawelbaran** [comment]
@BHoMBot check dataset-compliance
@BHoMBot check copyright-compliance
@BHoMBot check ready-to-merge

**Pull request merged**